frontends/concrete-python/examples/levenshtein_distance/README.md

```diff
@@ -131,33 +131,31 @@ Successful end
 - `python levenshtein_distance.py --autoperf`: Benchmark with random strings, for the different alphabets.
 
 ```
-
 Typical performances for alphabet ACTG, with string of maximal length:
 
```

Comment:
why those times are worst than before?

Reply:
even for a fixed CP and a fixed commit, I see a lot of variability in the perfs, on my computer: like

```
Typical performances for alphabet string, with string of maximal length:

    Computing Levenshtein between strings 'agbu' and 'ypaq' - OK in 17.57 seconds
    Computing Levenshtein between strings 'wvfo' and 'yjdh' - OK in 17.51 seconds
    Computing Levenshtein between strings 'imry' and 'nmwh' - OK in 17.88 seconds
```

vs

```
        Typical performances for alphabet string, with string of maximal length:

            Computing Levenshtein between strings 'rjba' and 'buet' - OK in 20.90 seconds
            Computing Levenshtein between strings 'vzcf' and 'oyld' - OK in 20.99 seconds
            Computing Levenshtein between strings 'bamq' and 'lvfy' - OK in 19.06 seconds
```

Reply:
or 

```
    Computing Levenshtein between strings 'wznn' and 'ljmi' - OK in 17.83 seconds
    Computing Levenshtein between strings 'pkjp' and 'zwbw' - OK in 16.65 seconds
    Computing Levenshtein between strings 'jjwf' and 'muxe' - OK in 16.68 seconds
```

vs

```
    Computing Levenshtein between strings 'nbnz' and 'mgel' - OK in 30.92 seconds
    Computing Levenshtein between strings 'oaaw' and 'thwv' - OK in 29.52 seconds
    Computing Levenshtein between strings 'mrcw' and 'ynxc' - OK in 29.72 seconds
```

Reply:
(with same SHA1 of sources)

Reply:
Made https://github.com/zama-ai/concrete-internal/issues/827